locksmith/src/operations/metadataOperations.ts

```diff
@@ -138,3 +149,32 @@ const defaultMappings = (address: string, host: string, keyId: string) => {
   defaultResponse.description = `${defaultResponse.description} Unlock is a protocol for memberships. https://unlock-protocol.com/`
   return defaultResponse
 }
+
+export const isKeyOwnerOrLockVerifier = async ({
```

Comment:
Should this not take into account lock manager as well like the original API?

Afaik, we don't automatically create lock manager as verifier on the backend.

Reply:
I'm saying it should also allow loco manager to access the data.

Reply:
Oh, get it! 
added also check for lock manager, thk!